onboarding.md

```diff
@@ -0,0 +1,39 @@
+### Onboarding
+Congratulations and welcome to voiio!
+
+We are happy to have you in the team, we know that starting a new job can be both exciting and a little bit daunting. This guide is meant to help you get settled.
+
+### Checklist
+To make sure you get a smooth onboarding and enjoy it, your buddy will be there to help you accomplish the following checklist.
```

Comment:
You are referencing the buddy here, without prior mentioning it. Maybe you want to add a section about the buddy first?